docs/reference/commandline/run.md

```diff
@@ -330,6 +330,13 @@ This adds the `busybox` container to the `mynet` network.
 $ docker run -itd --net=my-multihost-network busybox
 ```
 
+You can also choose the IP addresses for the container with `--ip` and `--ip6`
+flags when you start the container on a user-defined network.
+
+```bash
+$ docker run -itd --net=my-multihost-network --ip=10.10.9.75 busybox
```

Comment:
Actually if we do not start it with that option set, given no command is specified after the image name, the container will go in stopped state right away and user won't be able to inspect a running container, checking the settings while replicating the documentation examples as tutorial.